patches/server/0957-don-t-warn-about-offline-mode.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Mon, 2 Jan 2023 10:35:23 +0100
+Subject: [PATCH] don't warn about offline mode
+
+
+diff --git a/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java b/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
+index 51b3db0b6c2cede95b584268e035c0fb36d38094..95d5c4ab5610d3b5b4cc7ce1fbe6a75031888c5b 100644
+--- a/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
++++ b/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
+@@ -1,5 +1,6 @@
+ package net.minecraft.server.dedicated;
+ 
++import com.destroystokyo.paper.PaperConfig;
+ import com.google.common.collect.Lists;
+ import com.mojang.authlib.GameProfile;
+ import com.mojang.datafixers.DataFixer;
+@@ -19,6 +20,8 @@ import java.util.Locale;
+ import java.util.Optional;
+ import java.util.function.BooleanSupplier;
+ import javax.annotation.Nullable;
++
++import io.papermc.paper.configuration.GlobalConfiguration;
```

Comment:
This can be better a inline-import.

Reply:
okay, but why?

Reply:
well this or add the // Paper and remove the new line.. in the most of cases its better a patch with less diffs for this its better the inline import.